src/utils/api.test.js

```diff
@@ -37,10 +37,11 @@ describe('fetchCall', () => {
 
   it('should throw an error if response not ok', async () => {
     //setup
-    const expectedError = Error('Error fetching, code: 404')
+    const expectedError = Error('Error fetching, code: 404, Could not Post')
```

Comment:
@dylhof I think we talked about this change. I am not entirely sure. Does this look ok to you?

Reply:
Yeah, this looks good. Thanks for checking 👍